jsk_pcl_ros/include/jsk_pcl_ros/color_filter.h

```diff
@@ -82,6 +82,7 @@ namespace jsk_pcl_ros
     virtual void unsubscribe();
     boost::shared_ptr<message_filters::Synchronizer<SyncPolicy> >sync_;
 
+    bool keep_organized_;
```

Comment:
Does `keep_organized_` need to be the attribute of class?
It seems only used in `onInit` function.